src/test/java/com/artipie/test/TestContainer.java

```diff
@@ -88,7 +90,32 @@ public String execStdErr(final String... command) throws Exception {
      * @throws Exception In case of exception during execution command in container.
      */
     public String execStdout(final String... command) throws Exception {
-        return this.exec(command).getStdout().replace("\n", "");
+        final Container.ExecResult exec = this.exec(command);
+        final int code = exec.getExitCode();
+        if (code != 0) {
+            this.logStd(exec, true);
+            throw new IllegalStateException(
+                String.format(
+                    "'%s' failed with %s code", String.join(" ", command), code
+                )
+            );
+        }
+        this.logStd(exec, false);
+        return exec.getStdout().replace("\n", "");
```

Comment:
@genryxy the code for replacing `\n` with empty string is duplicated 3 times in this class, I'd suggest to extract a function for that